config.yaml

```diff
@@ -166,10 +166,10 @@ options:
       the Juju version is dynamically retrieved.
     type: string
     default:
-  jem-url:
+  bundleservice-url:
     type: string
     default: ''
```

Comment:
I concur with @hatched here--as bundleservice isn't an optional value like jem, it would be nice if the config were explicit, rather than relying on the implicit default hidden in the gui code.